darity.go

```diff
@@ -91,6 +101,27 @@ func (c *Client) APIVersion() (int, error) {
 	return c.ioctl(c.kvm.Fd(), kvmGetAPIVersion, 0)
 }
 
+// CreateVM returns a VM struct built around the fd provided
+// by kvm.
+func (c *Client) CreateVM(t MachineType) (*VM, error) {
+	v, err := c.ioctl(c.kvm.Fd(), kvmCreateVM, 0)
+	if err != nil {
+		return nil, err
+	}
+	return &VM{
+		fd:    v,
+		ioctl: c.ioctl,
```

Comment:
I think we should actually add `*Client` to this struct instead of an `ioctlFunc`, since the client already has an open file handle on the KVM device.